server-timing/load.php

```diff
@@ -10,6 +10,9 @@
 	exit; // Exit if accessed directly.
 }
 
+define( 'PERFLAB_SERVER_TIMING_SETTING', 'perflab_server_timing_settings' );
+define( 'PERFLAB_SERVER_TIMING_SCREEN', 'perflab-server-timing' );
```

Comment:
Would it make sense to start using `const`? We get better typing.

```suggestion
const PERFLAB_SERVER_TIMING_SETTING = 'perflab_server_timing_settings';
const PERFLAB_SERVER_TIMING_SCREEN = 'perflab-server-timing';
```

PhpStorm complains about the use of `define()`. Using `const` was introduced in PHP 5.3.

Reply:
Hmm, not really common in WP land I guess? I don't feel strongly about it, but we use `define` everywhere else in PL, so I think this should rather be holistically discussed (and potentially changed) in another issue.

Reply:
Sounds good